pages/common/csvformat.md

```diff
@@ -0,0 +1,24 @@
+# csvformat
+
+> Convert a CSV file to a custom output format.
+> Included in csvkit.
+
+- Convert to a tab-delimited file:
+
+`csvformat -T {{data.csv}}`
+
+- Convert delimiters to a custom character:
+
+`csvformat -D "{{custom_character}}" {{data.csv}}`
+
+- Convert line endings to carriage return + line feed:
+
+`csvformat -M "{{\r\n}}" {{data.csv}}`
```

Comment:
Good point -- we could add that to the description, e.g. "carriage return (^M) + line feed" even if that's not the original meaning, as it serves as a nice mnemonic :)